specification/logs/events/semantic_conventions/browser.md

```diff
@@ -0,0 +1,165 @@
+# Semantic conventions for browser events
+
+**Status**: [Experimental](../../document-status.md)
+
+
+<details>
+<summary>Table of Contents</summary>
+<!-- Re-generate TOC with `markdown-toc --no-first-h1 -i` -->
+
+<!-- toc -->
+
+- [Page View](#page-view)
+- [Ajax](#ajax)
+- [Exception](#exception)
+- [UserAction](#useraction)
+- [PageNavigationTiming](#pagenavigationtiming)
+- [AjaxTiming](#ajaxtiming)
+- [ResourceTiming](#resourcetiming)
+
+<!-- tocstop -->
+
+</details>
+
+
+This document describes the semantic conventions for browser events. The events may be represented either as Span Events or Log Events.
+
+All events have the following three high-level attributes. The domain has a fixed value of `browser`. The event name is specified at the beginning of each section below. The payload of the event goes in an attribute called `event.data` and the fields described below for each event go into this attribute.
+
+| Key  | Value | Description  |
+|---|---|---|
+| `event.domain` | browser | Fixed value |
+| `event.name` | | Described in each section below|
```

Comment:
I think if we can keep this table format consistent with the other tables, it will be great.